### PR TITLE
Add examples

### PR DESCRIPTION
**Pull request by ayinloya**
Adds examples
Adds the glitch project found [here](https://glitch.com/edit/#!/smart-camera-web-demo-node) as sample express project

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /node_modules
-/.nyc_output
\ No newline at end of file
+/.nyc_output
+.env
+examples/example-project/node_modules/
```
Comment (mldangelo) [inline .gitignore:4]: You shouldn't need to list node_modules twice. Consider switching this file out with https://github.com/github/gitignore/blob/main/Node.gitignore  You should also add `.DS_Store` and remove the one you committed. 
```suggestion
.DS_Store
```

File: examples/biometric_kyc.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-len */
```
Comment (mldangelo) [inline examples/biometric_kyc.js:1]: Don't disable this. Conform to the rule instead.

File: examples/example-project/LICENSE [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
+
+******************************************************************************
+
+THIRD-PARTY SOFTWARE
+
+
+1. express: Express is a web framework focused on developer experience.
```
Comment (mldangelo) [inline examples/example-project/LICENSE:8]: Express is MIT licensed. You should not need to copy the express license into this project.

File: examples/example-project/sample.env [hunk 4]
```diff
@@ -0,0 +1,4 @@
+NODE_ENV=development
+PARTNER_ID=
+API_KEY=
+SID_SERVER=0
```
Comment (mldangelo) [inline examples/example-project/sample.env:4]:
```suggestion
SID_SERVER=0
```

File: examples/example-project/server.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+const express = require('express');
+const { v4: UUID } = require('uuid');
+
+if (process.env.NODE_ENV === 'development') {
+  // eslint-disable-next-line global-require
+  const dotenv = require('dotenv');
+
+  dotenv.config();
+}
```
Comment (mldangelo) [inline examples/example-project/server.js:11]: why not just  ?
```suggestion
require('dotenv').config()
```

**Review by mldangelo — COMMENTED** [review]
Hi, this is great. Can you please document the examples in a readme file? You should also think about adding a package.json in the examples folder. Take a look at any one of the next examples (see [here](https://github.com/vercel/next.js/tree/canary/examples/api-routes-cors)) for inspiration.

**Review by mldangelo — APPROVED** [review]

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,105 @@
...
+.next
+out
+
+# Nuxt.js build / generate output
+.nuxt
+dist
+
+# Gatsby files
+.cache/
+# Comment in the public line in if your project uses Gatsby and not Next.js
```
Comment (okufoniyi) [inline .gitignore:84]: Typo *in if* # Comment in the public line if your project uses Gatsby and not Next.js

Comment (mldangelo) [reply .gitignore:84]: This comes from https://github.com/github/gitignore/blob/main/Node.gitignore#L96 It's in the official github .gitignore. I think we can ignore it.

**Pull request merged**